crates/ruff_python_parser/tests/snapshots/invalid_syntax@clause_expect_single_statement.py.snap

```diff
@@ -0,0 +1,53 @@
+---
+source: crates/ruff_python_parser/tests/fixtures.rs
+input_file: crates/ruff_python_parser/resources/inline/err/clause_expect_single_statement.py
+---
+## AST
+
+```
+Module(
+    ModModule {
+        range: 0..23,
+        body: [
+            If(
+                StmtIf {
+                    range: 0..8,
+                    test: BooleanLiteral(
+                        ExprBooleanLiteral {
+                            range: 3..7,
+                            value: true,
+                        },
+                    ),
+                    body: [],
+                    elif_else_clauses: [],
+                },
+            ),
+            If(
+                StmtIf {
+                    range: 9..22,
+                    test: BooleanLiteral(
+                        ExprBooleanLiteral {
+                            range: 12..16,
+                            value: true,
+                        },
+                    ),
+                    body: [
+                        Pass(
+                            StmtPass {
+                                range: 18..22,
+                            },
+                        ),
+                    ],
+                    elif_else_clauses: [],
+                },
+            ),
+        ],
+    },
+)
+```
+## Errors
+
+  |
+1 | if True: if True: pass
+  |          ^^ Syntax Error: Expected a single statement
```

Comment:
Nit: I think the message here might be confusing because what follows after the `:` is a single statement. Should it be *simple* statement or should we invert it and say that compound statements aren't allowed?

Reply:
Oh, it should be simple statement. This branch is taken if there's no NEWLINE token, so we can't expect a compound statement.